website/_posts/blog/2018-10-30-introduce-data-source-sdk-v2.6.0.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post-blog
+title:  Introduce data source SDK
+date:   2018-10-30 20:00:00
+author: Youcheng Zhang
+categories: blog
+---
+
+## Data source SDK
+
+Apache Kylin has already supported several data sources like vertica, SQL Server through JDBC. But we found that it takes much efforts to develop an implementation to a new source engine, like supporting metadata sync, cube build and query pushdown. It’s mainly because the SQL dialects and jdbc implementations between source engines are quite different.
```

Comment:
"vertica" need be "Vertica" or "HP Vertica" (V in uppercase); Besides, I'm not sure whether Kylin supports Vertica or not because there is not document, I suggest change it to "Amazon Redshift".